scripts/google/covid_harvard/StatisticalVariable.mcf

```diff
@@ -0,0 +1,27 @@
+Node: dcid:HarvardCOVID19CumulativeCases
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalCondition
+statType: dcs:measuredValue
+measuredProperty: dcs:cumulativeCount
+medicalStatus: dcs:ConfirmedCase
+
+Node: dcid:HarvardCOVID19CumulativeDeaths
```

Comment:
We will finalize the StatVar ID convention tomorrow, but let's use what Ian has now: mmethod_statType_measuredProperty_popType_v1_v2...

So here,

<your mmethod decision>_CumulativeCount_MedicalCondition_COVID_19_PatientDeceased